### PR TITLE
Modified image zoom

### LINKED ISSUE #3281: Pan&zoom, Invert-image, image-mouse-clicks toggles pan&zoom off. (camera-catalogue)
Related page: [panthera-research/camera-catalogue/classify](https://www.zooniverse.org/projects/panthera-research/camera-catalogue/classify)

After using the invert-image option on the page, the pan&zoom feature will consistently toggle itself Off (*if On*) on mouse-click on the image (*blocking any further mouse related image panning*).
-- *Limited to the duration of the current classification subject.*
-- *Other pan&zoom inputs not effected.*

###### _Local stuff: Chrome(49), Desktop, Windows(xp), 1024x768._

### PR DESCRIPTION
**Pull request by eatyourgreens**
Fixes #3281 .

Describe your changes.
Removes the `ModifiedImage` component and moves the SVG invert filter into the `SVGImage` component.
Fixes mouse drag not working for `ModifiedImage` by replacing it with `SVGImage` in the subject viewer.

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] Did you deploy a staging branch? https://modified-image-zoom.pfe-preview.zooniverse.org/
- [ ] Did you get any type checking errors from [Babel Typecheck](https://github.com/codemix/babel-plugin-typecheck)

## Optional

- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?

### CODE REVIEW

**Comment by wgranger** [comment]
Ha. I was actually in the process of making the same changes on a different branch when I saw this PR open. I think this works well for now since there is only one filter (invert). If we decide to add more filters to PFE in the future, it might be good to create a filters file in the lib that holds everything so `SVGImage` doesn't get bloated.

File: app/components/svg-image.jsx [hunk 1]
```diff
@@ -1,4 +1,21 @@
-import React, {PropTypes} from 'react';
+import React from 'react';
+
+const FILTERS = {
+    invert: "url('#svg-invert-filter')",
+}
+
+const INVERT =
+  '<svg style="position: fixed; right: 100%; top: 100%; visibility: hidden;">\
```
Comment (wgranger) [inline app/components/svg-image.jsx]: Lint is complaining `Multiline support is limited to browsers supporting ES5 only.`, which might not be a big deal. Backticks will remove this warning, however.

File: app/components/svg-image.jsx [hunk 2]
```diff
@@ -1,4 +1,21 @@
-import React, {PropTypes} from 'react';
+import React from 'react';
+
+const FILTERS = {
+    invert: "url('#svg-invert-filter')",
+}
```
Comment (wgranger) [inline app/components/svg-image.jsx]: Missing semicolon at the end of this constant. Also, line four is indented too much.

File: app/components/svg-image.jsx (hunk 1 above)
Comment (eatyourgreens) [inline app/components/svg-image.jsx]: Ah, right. I didn't realise the syntax had changed for multiline strings.

**Comment by eatyourgreens** [comment]
@wgranger ESLint should be happier now, and the tests should stop warning about missing required props on SVGImage.

**Comment by wgranger** [comment]
LGTM

**Pull request merged**